Sources/Plasma/PubUtilLib/plPhysX/plSimulationMgr.cpp

```diff
@@ -292,11 +293,20 @@ plSimulationMgr::plSimulationMgr()
 bool plSimulationMgr::InitSimulation()
 {
     fSDK = NxCreatePhysicsSDK(NX_PHYSICS_SDK_VERSION, NULL, &gErrorStream);
-    if (!fSDK)
+    if (!fSDK) {
+        fLog->AddLine("Phailed to init PhysX SDK");
         return false; // client will handle this and ask user to install
+    }
 
     fLog = plStatusLogMgr::GetInstance().CreateStatusLog(40, "Simulation.log", plStatusLog::kFilledBackground | plStatusLog::kAlignToTop);
-    fLog->AddLine("Initialized simulation mgr");
+    fLog->AddLine("Initialized PhysX SDK");
+
+    if (!NxInitCooking(nullptr, &gErrorStream)) {
```

Comment:
Does the normal PhysX client software include the cooking SDK libraries?  If not, can they be redistributed?